tests/nnstreamer_filter_ncnn/runTest.sh

```diff
@@ -100,7 +100,7 @@ function run_pipeline() {
 # Property reading test for accelerator
 run_pipeline true:gpu
 cat info | grep "accl = gpu$"
-testResult $? 8 "vulkan accelerator test" 0 1
+testResult $? 2 "vulkan accelerator test" 0 1
```

Comment:
Let's check the return value of ```run_pipeline``` as well.

Reply:
If I check ```$?``` after line 101, it fails.

Reply:
FYI, the failed one is the test on the top and it actually didn't make the ```ncnn.out.log``` or fail to write the text into the file.